### DISCUSSION TITLE
How to use i18n for folder name?

### DISCUSSION THREAD

**Discussion by allentown521** [fuma-nama/fumadocs#1150] [Q&A]
I've searched the documentation and didn't find a setting to internationalize the names of the folders

**Comment by fuma-nama** [comment]
You can add locale code to meta file

**Reply by allentown521** [reply]
Are there any examples? As far as I know, meta is just a json file, I tried setting it up like this and it had no effect
![image](https://github.com/user-attachments/assets/f35181eb-1447-4477-9aab-3de3f4d29872)

**Comment by allentown521** [comment]
I've found that option. I suggest adding also a chapter on internationalization, it should belong there too
https://fumadocs.vercel.app/docs/ui/page-conventions#internationalization

**Reply by fuma-nama** [reply]
It already mentioned, please read the docs

**Comment by allentown521** [comment]
But there is still a question, how to internationalize "folder group" name?

**Comment by wztlink1013** [comment]
@allentown521 I would like to ask, is this how the code is organized for folder internationalization? I organized the code this way but it didn't work. 

![image](https://github.com/user-attachments/assets/35865c56-67db-4a0f-a9dc-747dc4068d7e)

**Reply by fuma-nama** [reply]
Hmm then it's probably a bug, will look into that

**Reply by fuma-nama** [reply]
Can't seems to reproduce it, need more information
